config.h.in.cmake

```diff
@@ -151,6 +151,9 @@
 /* Disable cl_khr_fp16 because fp16 is not supported */
 #cmakedefine _CL_DISABLE_HALF
 
+/* Disable cl_khr_fp64 because fp64 is not supported */
```

Comment:
Does it disable it for all devices? What about heterogeneous platforms where host does not support fp64 but the co-processor (GPU) does?